flow/embedded_views.h

```diff
@@ -85,6 +98,8 @@ class Mutator {
         return matrix_ == other.matrix_;
       case opacity:
         return alpha_ == other.alpha_;
+      case backdrop_filter:
+        return *filter_ == *other.filter_;
```

Comment:
This seems like it could result in some very strange results, since `DlImageFilter` does not implement `operator==`.

@flar might have thoughts on whether it's feasible/a good idea to implement operator== there. It seems like it shouldn't be terribly hard, and we should just do it.

Reply:
Ahh perfect, thanks!